geomesa-process/geomesa-process-vector/src/main/scala/org/locationtech/geomesa/process/tube/TubeBuilder.scala

```diff
@@ -156,6 +156,56 @@ class LineGapFill(tubeFeatures: SimpleFeatureCollection,
 
   def nextId: String = id.getAndIncrement.toString
 
+  /**
+    * This function checks if a segment crosses the IDL.
+    * @param point1 The first point in the segment
+    * @param point2 The second point in the segment
+    * @return boolean representing validity of a segment based on segment length
+    */
+  def crossesIDL(point1:Coordinate,point2:Coordinate): Boolean = {
+    return (Math.abs(point1.x-point2.x) > 350)
```

Comment:
this seems somewhat arbitrary... should we maybe allow it to be passed into the tubeselect process?

Reply:
You are correct. That was arbitrary. I updated the way that these IDL crossings are detected.